tox.ini

```diff
@@ -1,11 +1,11 @@
 [tox]
-envlist = py27,pypy,py34,py35,py36,py27-zodb,pypy-zodb,py27-pure-zodb,docs
+envlist = py27,pypy,py34,py35,py36,py27-zodb,pypy-zodb,py27-pure-zodb,docs,coverage
 
 [testenv]
 commands =
     zope-testrunner --test-path=src []
 deps =
-    .[test]
+    .[test,docs]
```

Comment:
D'oh. They don't anymore. That's a leftover from when I was trying to run all doctests with Sphinx. Thanks!